baseapp/baseapp.go

```diff
@@ -364,7 +333,13 @@ func handleQueryApp(app *BaseApp, path []string, req abci.RequestQuery) (res abc
 		default:
 			result = sdk.ErrUnknownRequest(fmt.Sprintf("Unknown query: %s", path)).Result()
 		}
-		value := app.cdc.MustMarshalBinary(result)
+
+		// Encode with json
+		value, err := json.Marshal(result)
```

Comment:
Hmm - won't this marshal e.g. numbers different than Amino JSON? Maybe we should use an empty Amino codec (just for the custom JSON) to avoid returning different JSON formats in different places?